src/main/java/org/testng/internal/BshMock.java

```diff
@@ -1,12 +1,15 @@
 package org.testng.internal;
 
 import org.testng.ITestNGMethod;
+import org.testng.TestNGException;
 
 public class BshMock implements IBsh {
 
   @Override
   public boolean includeMethodFromExpression(String expression, ITestNGMethod tm) {
-    return false;
+    String msg = "Beanshell related classes could not be found in CLASSPATH."
+        + "Please add a dependency to bsh.Interpreter (beanshell jar) to your build file and try again.";
```

Comment:
Not sure about providing the GAV here because it is already done in build files and interpreted by build tools.